flathunter/config.py

```diff
@@ -32,6 +32,11 @@ def __getitem__(self, value):
     def get(self, key, value=None):
         return self.config.get(key, value)
 
+    def database_location(self):
```

Comment:
Prefer config functions to directly accessing the keys - this gives us more flexibility to calculate values.